base_delivery_carrier_files/__manifest__.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2012 Camptocamp SA
+# Author: Guewen Baconnier
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    "name": "Base Delivery Carrier Files",
+    "version": "14.0.1.0.0",
+    "category": "Generic Modules/Warehouse",
+    "author": "Camptocamp,Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "website": "https://github.com/OCA/delivery-carrier",
+    "depends": ["base", "stock", "delivery"],
```

Comment:
in v14 delivery implies stock
delivery -> sale_stock ->  stock_account -> stock